packages/core/src/electron-main/electron-main-application.ts

```diff
@@ -445,26 +444,6 @@ export class ElectronMainApplication {
         ).sort().join('-');
     }
 
-    /**
-     * Catch certain keybindings to prevent reloading the window using keyboard shortcuts.
-     */
-    protected attachGlobalShortcuts(electronWindow: BrowserWindow): void {
-        const handler = this.config.electron?.disallowReloadKeybinding
-            ? () => { }
-            : () => this.reload(electronWindow);
-        const accelerators = ['CmdOrCtrl+R', 'F5'];
```

Comment:
In order to have the same feature parity as we had with these changes, I'd try and bring <kbd>F5</kbd> back as an possible alternative to <kbd>ctrlcmd</kbd>+<kbd>r</kbd>.

Reply:
On second thought <kbd>F5</kbd> is used to start a debug session (similarly to vscode) so we can probably exclude it as a possibility to reload.